src/main/java/com/lacunasoftware/pkiexpress/DataSigner.java

```diff
@@ -0,0 +1,47 @@
+package com.lacunasoftware.pkiexpress;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+
+public class DataSigner extends Signer {
+    
+    private String toSignData;
+
+    public String getToSignData() {
+        return this.toSignData;
+    }
+    
+    public void setToSignData(String toSignData) {
+        this.toSignData = toSignData;
+    }
+
+    public DataSigner(PkiExpressConfig config) {
+		super(config);
+	}
```

Comment:
There are some inconsistencies in the identation of this file (probably a mix of tabs and spaces). Please use only tabs in this project